Source/WebCore/loader/ContentFilter.cpp

```diff
@@ -363,6 +363,17 @@ void ContentFilter::deliverStoredResourceData()
     m_buffers.clear();
 }
 
+#if HAVE(AUDIT_TOKEN)
+void ContentFilter::setHostProcessAuditToken(std::optional<audit_token_t>&& token)
+{
+    if (!token)
+        return;
```

Comment:
If we had previously set a token on `m_contentFilters` and now have `nullopt`, won't the filters be left with a stale audit token?